rinja_derive/src/generator/expr.rs

```diff
@@ -256,25 +257,22 @@ impl<'a> Generator<'a, '_> {
             "format" => Self::_visit_format_filter,
             "join" => Self::_visit_join_filter,
             "json" | "tojson" => Self::_visit_json_filter,
-            "linebreaks" => Self::_visit_linebreaks_filter,
-            "linebreaksbr" => Self::_visit_linebreaksbr_filter,
-            "paragraphbreaks" => Self::_visit_paragraphbreaks_filter,
+            "linebreaks" | "linebreaksbr" | "paragraphbreaks" => Self::_visit_linebreaks_filters,
             "pluralize" => Self::_visit_pluralize_filter,
             "ref" => Self::_visit_ref_filter,
             "safe" => Self::_visit_safe_filter,
-            "urlencode" => Self::_visit_urlencode_filter,
-            "urlencode_strict" => Self::_visit_urlencode_strict_filter,
-            "value" => return self._visit_value(ctx, buf, args, generics, node, "`value` filter"),
-            name if BUILTIN_FILTERS.contains(&name) => {
-                return self._visit_builtin_filter(ctx, buf, name, args, generics, node);
+            "urlencode" | "urlencode_strict" => Self::_visit_urlencode_filter,
+            name if BUILTIN_FILTERS.contains(&name) => Self::_visit_builtin_filter,
+            "value" => {
+                return self._visit_value(ctx, buf, args, generics, node, "`value` filter");
+            }
+            _ => {
+                return self._visit_custom_filter(ctx, buf, name, args, generics);
             }
```

Comment:
```suggestion
            "value" => {
                return self._visit_value(ctx, buf, args, generics, node, "`value` filter");
            }
            name => if BUILTIN_FILTERS.contains(&name) {
                Self::_visit_builtin_filter
            } else {
                return self._visit_custom_filter(ctx, buf, name, args, generics);
            },
```

This allows to remove one arm and have the `contains` call to be done after all other strings have been checked.